packages/node-cli/messages.json

```diff
@@ -52,7 +52,7 @@
 	"COMMAND_DEPLOY_QUESTIONS_CHOICES_ACCOUNT_SPECIFIC_VALUES_DISPLAY_WARNING": "Display a warning",
 	"COMMAND_DEPLOY_QUESTIONS_PERFORM_LOCAL_VALIDATION": "Do you want to validate locally before deploying?",
 	"COMMAND_DEPLOY_MESSAGES_APPLYING_CONTENT_PROTECTION": "The \"{0}\" project has been deployed applying content protection.",
-	"COMMAND_DEPLOY_MESSAGES_DEPLOYING": "Deploying using authentication ID (authID) {0}.",
+	"COMMAND_DEPLOY_MESSAGES_DEPLOYING": "Deploying project \"{0}\" with \"{1}\" authID",
```

Comment:
Looks good but can we change the order to?

Deploying project "myproject" with authID "myauthid"

Same for validation message